ocs_ci/ocs/resources/ocs.py

```diff
@@ -349,3 +352,32 @@ def ocs_install_verification(timeout=600, skip_osd_distribution_check=False):
     assert {defaults.CEPHFS_PROVISIONER, defaults.RBD_PROVISIONER} == (
         {item['metadata']['name'] for item in csi_driver.get()['items']}
     )
+
+    # Verify ceph osd tree have device set PVC names specified
+    log.info("Checking for device set PVC names in ceph osd tree output.")
+    deviceset_pvcs = [
```

Comment:
please use ocs_ci.ocs.resources.pvc.get_deviceset_pvs

Reply:
get_deviceset_pvs is giving list of PV. Names of PVCs are required here.

Reply:
got you. Maybe would be good to create a function for this implementation in pvc.py

Reply:
Added new function get_deviceset_pvcs in pvc.py